hamlet-cli/hamlet/command/common/decorators.py

```diff
@@ -40,6 +40,11 @@ def wrapper(ctx, *args, **kwargs):
 def common_district_options(f):
     """Common options for district config"""
 
+    @click.option(
+        "--root-dir",
+        envvar="ROOT_DIR",
+        help="The root CMDB directory",
```

Comment:
This is only necessary when either ROOT_DIR is unset or the root.json file isn't going to be found - should the Help message indicate its an override and not necessary for most cases?

Reply:
Added note that they are all based on the current CMDB location.